ibm_catalog.json

```diff
@@ -5,10 +5,201 @@
+                    "architecture": {
+                        "features": [
+                            {
+                                "title": "Separate VPC for management",
+                                "description": "Yes"
+                            },
+                            {
+                                "title": "Separate VPC for Internet Services",
+                                "description": "Yes"
+                            },
+                            {
+                                "title": "Separate VPC for Private Services",
+                                "description": "Yes"
+                            },
+                            {
+                                "title": "1 Virtual Server Instances for every VPC",
+                                "description": "Yes"
+                            },
+                            {
+                                "title": "Increases security with Key Management",
+                                "description": "Yes"
+                            },
+                            {
+                                "description": "PowerVS Workspace with all required components",
+                                "title": "Power Virtual Server Workspace"
+                            },
+                            {
+                                "description": "Proxy service to reach public internet from PowerVS Workspace",
+                                "title": "Internet proxy on VPC"
+                            },
+                            {
+                                "description": "NFS server, NTP forwarder, and DNS forwarder reachable from PowerVS Workspace",
+                                "title": "Additional management configurations on VPC"
+                            }
+                        ],
+                        "diagrams": [
+                            {
+                                "diagram": {
+                                    "caption": "Power infrastructure for deployable architectures",
+                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/main/reference-architectures/full-stack/deploy-arch-ibm-pvs-inf-full-stack.svg",
+                                    "type": "image/svg+xml"
+                                },
+                                "description": "Full stack solution which deploys Secure infrastructure on VPC for regulated industries and deploys  Power Virtual Server workspace and connects it with VPC services. Proxy service for public internet access from PowerVS workspace is configured. \n\nAdditional management components (NFS server, NTP forwarder, and DNS forwarder) may be installed on VPC optionally."
+                            }
+                        ]
+                    }
+                },
+                {
+                    "label": "PowerVS Workspace Extension",
```

Comment:
@kbiegert - name is considered to be temporary. please inform if we get a switch between full-stack and extension variations